plugins/catalog-backend/config.d.ts

```diff
@@ -54,6 +54,23 @@ export interface Config {
      */
     readonly?: boolean;
 
+    /**
+     * Allow unknown type when create location.
+     *
+     * Setting 'locationService.create.allowUnknownType=false' forbids to create
+     * location with unknown type through location service.
+     * This is the default value.
+     *
+     * Setting 'locationService.create.allowUnknownType=true' allows to create
+     * location with unknown type through location service.
+     *
+     */
+    locationService?: {
```

Comment:
I'd like to drop the dependency on this being set in config, and keep it to code configuration if possible, as it's easier to evolve and deprecate over time.

You're still welcome to add it to config your side, and pull out the types and use that to pass to the builder, but that would be in your `packages/backend/src/` instead.